src/form/Form.js

```diff
@@ -0,0 +1,345 @@
+import './Form.css';
+import { useState, useEffect } from 'react';
+import DatePicker from 'react-datepicker';
+import { postHabit } from '../apiCalls';
+import flowerOne from '../images/flowers/flowerID_1.png';
+import flowerTwo from '../images/flowers/flowerID_2.png';
+import flowerThree from '../images/flowers/flowerID_3.png'
+import flowerFour from '../images/flowers/flowerID_4.png';
+
+function Form({ isActive, closeForm }) {
+    const [nameInput, setNameInput] = useState('')
```

Comment:
maybe for part 2 we can refactor the number of useStates, I've never used it but I think for the form we could implement useReducer() ?